### PR TITLE
allow using turbojpeg by pkg-config

### PR DESCRIPTION
**Pull request by igor-raits**
Signed-off-by: Igor Gnatenko <i.gnatenko.brain@gmail.com>

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -31,16 +31,20 @@ if (${STATIC_RUNTIME_LIBS})
...
-mark_as_advanced(TURBOJPEG_LIBRARIES TURBOJPEG_INCLUDEDIR)
-include_directories(${TURBOJPEG_INCLUDEDIR})
-link_directories(${TURBOJPEG_INCLUDEDIR})
-
 if (UNIX)
     include(FindPkgConfig)
 endif()
 
+if (UNIX)
+    pkg_check_modules(TURBOJPEG libjpeg)
```
Comment (Benau) [inline CMakeLists.txt]: Do you mean libturbojpeg here? otherwise travis build is broken

Comment (igor-raits) [reply CMakeLists.txt]: oh, right. forgot to push

**Comment by igor-raits** [comment]
@Benau pushed new version.

**Comment by igor-raits** [comment]
@Benau sorry, I have absolutely no idea how to make travis happy. I don't use Ubuntu myself, I use Fedora.

**Comment by deveee** [comment]
I had an interesting error. I have "dependencies" directory for windows compilation. But I didn't have libturbojpeg-dev installed for some reason (maybe it was removed after system upgrade).

So cmake found turbojpeg in "dependencies/include" and used ```include_directories(${TURBOJPEG_INCLUDEDIR})```

Then vpx used "dependencies/include" too and "vpx_codec_enc_init" failed to init because of ABI mismatch, which caused STK to crash.

So that +1 for using pkg-config ;)

**Comment by Benau** [comment]
kind of "hack" for ubuntu 
https://github.com/Benau/libopenglrecorder/commit/3a82d8c5cbf97baeb3fb4f638fa33a16d5c560da

TURBOJPEG_INCLUDEDIR cannot seem to be made find_path-able again if used by pkg-config

Good for you?

**Pull request merged**